src/data/map.js

```diff
@@ -1258,12 +1190,14 @@ const sqlifyIvFilter = (filter) => {
                     case 'S': column = 'sta_iv'; break;
                     case 'L': column = 'level';  break;
                     case 'CP': column = 'cp';    break;
+                    case 'GR': column = 'great_rank'; break;
+                    case 'UR': column = 'ultra_rank'; break;
```

Comment:
If you don't have plans to add support for filtering percentage or any other properties I think `GL` and `UL` would be better keywords personally.

Reply:
Agreed

Reply:
DiSaGrEeD

Reply:
Basically all mapping tools reference this as Great/Ultra League pokemon. The rank is just the text value. Changing to GL/UL just matches better. Agreed with versx & sketchysocks.

Users aren't filtering Great Rank Pokemon 1-3. They're filtering Great League Pokemon 1-3.

Reply:
Ok fine.